.eslintrc

```diff
@@ -1,5 +1,5 @@
 {
-  "parser": "babel-eslint",
+  // "parser": "babel-eslint",
```

Comment:
Does this cause issues when enabled? Slightly concerned we are disabling this here

Reply:
yes, editor goes crazy in `.ts(x)` files, as it still treats them as `js/flow`, reporting incorrect syntax.
It's the same issue as https://skyscanner.slack.com/archives/C03R95D4A0L/p1658992797736519

Reply:
Can I keep it disabled or do we want to look at a different solution?